### ISSUE TITLE
Publish via openbits.world

### ISSUE THREAD

**Issue by michael-spengler** [michael-spengler/decentralized-finance#3]
https://openbits.world/

This would be cool: 

![Screenshot 2021-01-20 at 00 39 17](https://user-images.githubusercontent.com/43786652/105107308-28cdf280-5ab8-11eb-9287-bb8c518ae1b7.png)

**Comment by michael-spengler** [comment]
--> 
```sh
openbits published-status decentralized-finance-defi@2.7.0
```

printed: 
![Screenshot 2021-01-20 at 00 46 10](https://user-images.githubusercontent.com/43786652/105107692-e822a900-5ab8-11eb-9946-381b03acb5ba.png)


--> publish seems to be successful

**Comment by michael-spengler** [comment]
this package can now also be installed via: 

```sh
openbits install decentralized-finance-defi 
```

https://openbits.world/#/explore-openbit/decentralized-finance-defi
